editor/src/components/canvas/dom-lookup.ts

```diff
@@ -636,3 +639,18 @@ export function canvasPointToWindowPoint(
     throw new Error('calling screenToElementCoordinates() before being mounted')
   }
 }
+
+export function canvasRectangleToWindowRectangle(
```

Comment:
Huh, I thought we already had this function

Reply:
me too :)